Earthfile

```diff
@@ -11,8 +11,11 @@ install-dind-script:
     SAVE ARTIFACT ./install-dind.sh
 
 test:
+    FROM alpine:3.18
     BUILD +test-install-dind-amd64
-    BUILD ./ssh+test-add-known-hosts
+    FOR dir IN "ssh git"
```

Comment:
nit: I would just use `BUILD ssh+test` and `BUILD git+test` directly since it's quicker.

Reply:
yeah I might have been trying to optimize too soon for having many directories

Reply:
changed